src/msha/handlers/dab.handler.spec.ts

```diff
@@ -0,0 +1,63 @@
+import http from "http";
+import { URL } from "url";
+import { injectHeaders, isDataApiRequest } from "./dab.handler";
+
+describe("isDataApiRequest", () => {
+  const req = { url: "/data-api/users" } as http.IncomingMessage;
+
+  it('returns true when the path starts with "/data-api/"', () => {
+    const result = isDataApiRequest(req);
+    expect(result).toEqual(true);
+  });
+
+  it('returns false when the path does not start with "/data-api/"', () => {
+    const req2 = { url: "/api/users" } as http.IncomingMessage;
```

Comment:
shall we add folllowing tests as well
a. `/data-api`  ( no slash )
b. `/xyz/data-api/users` ( data-api in between the url
c. `/DATA-aPi/` (ignore case comparison)

Reply:
added